snuba_sdk/conditions.py

```diff
@@ -142,7 +142,7 @@ class Or(BooleanCondition):
     conditions: ConditionGroup = field(default_factory=list)
 
 
-ConditionGroup = Sequence[Union[BooleanCondition, Condition]]
+ConditionGroup = List[Union[BooleanCondition, Condition]]
```

Comment:
If you're going to do this then please use `list` instead of `List`.

Reply:
Why is that? Using `list` produces mypy error: `list is not subscriptable`. That is because the expression `list[..]` is trying to subscript the object `list`

Reply:
You might have to add `from __future__ import annotations` to the top of the file. Using the lower case version is the default in newer versions of Python, so this is just making us forward compatible.

Reply:
Added, but it seems to be failing CI

Reply:
Using `list` only works for python > 3.9. We are still on 3.8 in prod so unfortunately we still need to use `List` from typing